ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__amd64-nightly.yaml

```diff
@@ -1203,6 +1203,19 @@ tests:
     test:
     - chain: openshift-e2e-test-qe
     workflow: cucushift-installer-rehearse-vsphere-upi-proxy
+- as: e2e-aws-ocm-osd-ccs-p1
```

Comment:
Please sort the jobs on this line.

Reply:
Strangely, the 'make update' does not help me do the sorting automatically. I will do it manually.